Makefile

```diff
@@ -0,0 +1,43 @@
+GO_EXEC := $(shell which go)
+GO_BIN := $(shell dirname $(GO_EXEC))
+GOROOT := $(shell dirname $(GO_BIN))
+
+DIST_DIR=./dist
+WASM_OUT=$(DIST_DIR)/ioticsIdentity.wasm
+WASM_EXEC_JS=./src/wasm_exec.js
+LIB_GO=./src/ioticsIdentity.go
```

Comment:
the golang src should be pulled from GH, by a specific tag.

Reply:
the advise is to use the wasm_exec.js of your local installation.